lightning/src/util/config.rs

```diff
@@ -126,6 +126,26 @@ pub struct ChannelHandshakeConfig {
 	///
 	/// [`KeysInterface::get_shutdown_scriptpubkey`]: crate::chain::keysinterface::KeysInterface::get_shutdown_scriptpubkey
 	pub commit_upfront_shutdown_pubkey: bool,
+
+	/// `their_channel_reserve_satoshis` is the minimum amount that the other node is to keep as a direct payment.
```

Comment:
I personally find it a bit confusing that this text starts by explaining another value than the one explicitly configured, even though it's related to the value that's configured. Would it make sense to start this text with something like:

Sets the proportion that of the `channel_value` that `their_channel_reserve_satoshis` is set to for both outgoing and incoming channels. `their_channel_reserve_satoshis` is the minimum amount ...